src/main/autoLaunch.js

```diff
@@ -0,0 +1,17 @@
+const AutoLaunch = require('auto-launch');
+
+async function upgradeAutoLaunch() {
+  if (process.platform === 'darwin') {
```

Comment:
Darwin is MacOS, right? What does this do on Linux platforms? If it's only supported on Windows, I wonder if a whitelist approach might be best here.

Reply:
Yeah, `darwin` means macOS. On Linux, this function should behave as well as Windows: Reconfigure auto-start for the current version.